data-loader/core/src/main/java/com/scalar/db/dataloader/core/dataimport/task/ImportTask.java

```diff
@@ -423,6 +430,20 @@ private boolean shouldFailForExistingData(
         && importOptions.getImportMode() == ImportMode.INSERT;
   }
 
+  /**
+   * Returns the given source record only if raw record logging is enabled in the import options.
+   *
+   * <p>This helper method centralizes the logic for conditionally including the raw source record
+   * in {@code ImportTargetResult}. If {@code logRawRecord} is disabled, {@code null} is returned to
+   * avoid storing or displaying the raw record.
+   *
+   * @param record the source record to include conditionally
+   * @return the provided record if raw record logging is enabled; otherwise {@code null}
+   */
+  private ObjectNode getRecordForLogging(ObjectNode record) {
```

Comment:
[nitpick] Should this method have `@Nullable` annotation for the return value?

Reply:
Thank you for the suggestion.
I have added the it in [541e9d1](https://github.com/scalar-labs/scalardb/pull/3189/commits/541e9d1139f21255b64358d20f08adcef5a0d4e2).